CHANGELOG.asciidoc

```diff
@@ -373,7 +373,7 @@ still supported. Users should switch to using `longFieldMaxLength`. If
 now use the `longFieldMaxLength` value.
 +
 Note that ultimately the maximum length of any tracing field is limited by the
-{apm-server-ref-v}/configuration-process.html#max_event_size[`max_event_size`]
+{apm-guide-ref}/configuration-process.html#max_event_size[`max_event_size`]
```

Comment:
https://github.com/elastic/observability-docs/issues/1385 suggests this one should be "Remove from Node.js Agent"?

Reply:
That was an incorrect assumption on my part. This config variable is still valid for the standalone method of running APM Server. We'll need to remove this at a later date.